src/main/java/gov/nasa/pds/harvest/search/crawler/actions/FileObjectRegistrationAction.java

```diff
@@ -298,10 +298,11 @@ private Metadata createFileObjectMetadata(FileObject fileObject,
     if (!fileObject.getAccessUrls().isEmpty()) {
       slots.add(new Slot(Constants.ACCESS_URLS, fileObject.getAccessUrls()));
     }
-    for (Iterator i = sourceMet.getHashtable().entrySet().iterator();
-    i.hasNext();) {
-      Map.Entry entry = (Map.Entry) i.next();
-      String key = entry.getKey().toString();
+    // for (Iterator i = sourceMet.getHashTable().entrySet().iterator();
```

Comment:
When considering commenting-out code, either:

- Include a comment explaining why it's commented out … or …  🤔
- If you can't come up with one, then just delete the code completely 😏